faithcomesbyhearing/dbp
# FCBH-378 Add book_id and iso notes on documentation

## Description
- Added book_id and iso notes on every endpoint that implement them

## Motivation and Context
- As a dev on using the dbp project, I’d like to have context on constants or linkages needed for requests.

## Issue Link

[FCBH-378](https://fullstacklabs.atlassian.net/browse/FCBH-378) 

## How Do I Test This?
- On your local environment run `http://dbp.test/open-api-v4.json` and verify that now the documentation have notes on the book_id and iso parameters

## Checklist:
